Documentation/tutorial/intro_toc.md

```diff
@@ -0,0 +1,99 @@
+
+#### Data model for `toc.yml`
+The data model for `toc.yml` is an **array** of *TOC Item Object*s.
+
+##### *TOC Item Object*
+*TOC Item Object* represents the data model for each *TOC Item*.
+
+Note that all the property names are **case sensitive**.
+
+Property Name | Type              | Description
+------------- | ----------------- | ---------------------------
+*name*        | string            | Specifies the title of the *TOC Item*.
+*href*        | string            | Specifies the hyperlink of the *TOC Item*.
+*homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
+*uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*. 
+*homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
+*items*       | *TOC Item Object* | Specifies the children *TOC Items*s of current *TOC Item*.
+
+Relative path in detail
+---------------
+If a *TOC Item* is linking to some relative path, there are three cases:
+
+1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
+2. Linking to a folder, which means, the value of the link ends with `/` explicitly, for example, `href: examples/`
+3. Linking to some local file.
+
+Each case is described in detail below.
+
+### Link to another *TOC File*
+If the *TOC Item* is linking to some other *TOC File*, it is considered as a placeholder of the referenced *TOC File*, and DocFX will extract content from that *TOC File* and insert into current *TOC Item* **recursively**. 
+
+This technique is always used when you want to combine several *TOC File*s into one single *TOC File*.
+
+### Link to a folder
+If the *Toc Item* is linking to a folder, ending with `/` explicitly, the link value for the *Toc Item* is determined in the following steps:
+1. If `homepage` or `homepageUid` is set, the link value is resolved to the relative path to `homepage`
+2. If `homepage` or `homepageUid` is not set, DocFX searches for *Toc File* under the folder, and get the first [relative link to local file](#link-to-local-file) as the link value for current *Toc Item*. For example, if the *Toc Item* is `href: article/`, and the content of `article/toc.yml` is as follows:
+
+    ```yaml
+    - name: Topic1
+      href: topic1.md
+    ```
+    The link value for the *Toc Item* is resolved to `article/topic1.md`.
+3. If there is no *Toc File* under the folder, the link value keeps unchanged.
+
+### Link to local file
+If the *Toc Item* is linking to a local file, we call this local file *In-Toc File*. This local file will have a system-generated property `_tocRel` equals to the relative path to the *Toc File*.
+
+Not-In-Toc Files
```

Comment:
give an example about the generated toc model and toc related metadata like tocrel and navrel